UIRes/serveropcode.json

```diff
@@ -1,14 +1,14 @@
 {
-  "ActorControlSelf": 615,
-  "HousingWardInfo": 221,
```

Comment:
Has HousingWardInfo been confirmed? According to <https://github.com/karashiiro/FFXIVOpcodes/>, it's `685` in dec, which is `0x2AD` in hex. That one doesn't get listed on the Ipcs.cs though, online in the decimal formatted json.

Reply:
Not sure, going to check, I was just looking in the .cs and didn't see it, didn't know it was in the json.

Reply:
Hmm, this looks fine? Not sure how else to test it though.
![image](https://user-images.githubusercontent.com/33433913/217891620-3ebff54c-e4e1-4105-b4cf-8429b90705e8.png)